app/js/controllers/relatedCtrl.js

```diff
@@ -3,6 +3,7 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		$scope.relatedOption = "relatedToEntry";
 		$scope.entryList = "";
 		$scope.playlistId = {id:'' ,text:''};
+		$scope.playerData.config.plugins["related"] = {disable: true};
```

Comment:
what if the related is enabled from before, don't you disable it?

Reply:
fixed